overlay/ipc_utils.c

```diff
@@ -0,0 +1,107 @@
+// Copyright 2007-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "ipc_utils.h"
+#include <string.h>
+
+#ifdef _WIN32
+#	include <direct.h>
+#	include <stdlib.h>
+#	include <malloc.h>
+#else
+#	include <pwd.h>
+#	include <stdio.h>
+#	include <stdlib.h>
+#	include <sys/stat.h>
+#	include <sys/types.h>
+#	include <unistd.h>
+#endif
+
+char *getRuntimePath__() {
+	size_t n   = 0;
+	char *path = NULL;
+
+#ifdef __linux__
```

Comment:
As far as i understood, they are defined by CMake. I use them since they are used in other bits of [C code in the overlay](https://github.com/mumble-voip/mumble/blob/master/overlay_gl/init_unix.c#L11).